d2common/d2resource/languages_map.go

```diff
@@ -49,3 +49,30 @@ func GetFontCharset(language string) string {
 
 	return charset[language]
 }
+
+// modificators for labels (used in string tables)
+// some of values need to be set up. For now values with "checked" comment
+// was tested and works fine in main menu.
+func getModifiers() map[string]int {
+	return map[string]int{
+		"ENG": 0, // (English) // checked
+		"ESP": 0, // (Spanish)
+		"DEU": 0, // (German) // checked
+		"FRA": 0, // (French)
+		"POR": 0, // (Portuguese)
+		"ITA": 0, // (Italian)
+		"JPN": 0, // (Japanese)
+		"KOR": 0, // (Korean)
+		"SIN": 0, //
+		"CHI": 0, // (Chinese)
+		"POL": 1, // (Polish) // checked
+		"RUS": 0, // (Russian) // checked
+	}
+}
+
+// GetLabelModifier returns modificator for language
+func GetLabelModifier(language string) int {
+	mod := getModifiers()
+
+	return mod[language]
+}
```

Comment:
replace all `modificator` with `modifier`

~~How are these integers used... ?~~ I think you should state that this table is used as an index offset for string keys of the form `#1234`

also, you could use just one function here:
```golang
// GetLabelModifier returns modificator for language
func GetLabelModifier(language string) int {
	modifiers := map[string]int{
		"ENG": 0, // (English) // checked
		"ESP": 0, // (Spanish)
		"DEU": 0, // (German) // checked
		"FRA": 0, // (French)
		"POR": 0, // (Portuguese)
		"ITA": 0, // (Italian)
		"JPN": 0, // (Japanese)
		"KOR": 0, // (Korean)
		"SIN": 0, //
		"CHI": 0, // (Chinese)
		"POL": 1, // (Polish) // checked
		"RUS": 0, // (Russian) // checked
	}

	return modifiers[language]
}
```